python/dgl/distributed/kvstore.py

```diff
@@ -953,7 +935,6 @@ def map_shared_data(self, partition_book):
             for _ in range(self._group_count-1):
                 response = rpc.recv_response()
                 assert response.msg == SEND_META_TO_BACKUP_MSG
-        self._freeze = True
```

Comment:
I think we cannot just simply remove the `freeze` code here. I will change the code in a new PR to fix it.